api/camunda/camunda-openapi.yaml

```
@@ -700,6 +754,58 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /decision-definitions/{decisionKey}/xml:
+    get:
+      tags:
+        - Decision Definition
```

```suggestion
        - Decision definition
```

category: casing

already taken care of in https://github.com/camunda/camunda/pull/22037